docs/student.md

```diff
@@ -86,6 +86,46 @@ The organizing committee does not recommend that students start research and dev
 
   - According to the final project evaluation results, the remaining project bonus will be paid in proportion.
 
+## Student bonus instruction
+
+1. How much bonus amount will each student receive?
+
+Each project will be awarded RMB 12000, RMB 9000 and RMB 6000 (before tax), which based on the level of project difficulty.
+
+2. When will the bonus be paid?
+
+Students who pass the mid-term assessment will receive 50% of the project bonus first. According to the final project evaluation results, the remaining project bonus will be paid in proportion. If the student fails the final evaluation, the rest bonus will not be paid.
+
+3. How to pay the students’ bonus?
+
+The students’ bonus will be paid by credit transfer method. There is no restriction on which bank but the bank account must be under your own name. Also, the organizing committee will collect the following information from the mentor:
+
+- Name
+
+- ID number
+
+- Bank account and branch information
+
+*Note: the organizing committee will take reasonable measures to protect private data, personal identity information and other sensitive information
+
+4. How to pay the foreign students’ bonus?
+
+- The foreign students’ bonus shall be settled in dollars, subject to the real-time exchange rate at the time of bonus payment.
+- The students’ bonus will be paid by credit transfer method. There is no restriction on which bank but the bank account must be under your own name. Also, the organizing committee will collect the following information from the mentor:
+  - Name and address of receiving bank
+  - Bank account
+  - Swiftcode
+  - Student’s name
+  - Student’s address
+  
+*Note: the above information is required according to the current international settlement policy. The organizing committee will take reasonable measures to protect private data, personal identity information and other sensitive information.
+
+The tax payment or tax payment certificate including the amount of this payment is applicable to the situation where Chinese and foreign personnel obtain income in China according to the tax terms, and individual income taxes withheld and paid by the organizing committee on behalf. The calculation method is as follows (unit: yuan)
```

Comment:
yuan should be Yuan. And I think we should note with Renminbi, RMB or CNY to explain Yuan. Ref: https://en.wikipedia.org/wiki/Renminbi